x-pack/plugins/ingest_pipelines/__jest__/client_integration/ingest_pipelines_list.test.ts

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { act } from 'react-dom/test-utils';
+
+import { API_BASE_PATH } from '../../common/constants';
+
+import { setupEnvironment, pageHelpers, nextTick } from './helpers';
+import { PipelineListTestBed } from './helpers/pipelines_list.helpers';
+
+const { setup } = pageHelpers.pipelinesList;
+
+jest.mock('ui/i18n', () => {
+  const I18nContext = ({ children }: any) => children;
+  return { I18nContext };
+});
+
+describe('<PipelinesList />', () => {
+  const { server, httpRequestsMockHelpers } = setupEnvironment();
+  let testBed: PipelineListTestBed;
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  describe('With pipelines', () => {
+    const pipeline1 = {
+      name: 'test_pipeline1',
+      description: 'test_pipeline1 description',
+      processors: [],
+    };
+
+    const pipeline2 = {
+      name: 'test_pipeline2',
+      description: 'test_pipeline2 description',
+      processors: [],
+    };
+
+    const pipelines = [pipeline1, pipeline2];
+
+    beforeEach(async () => {
+      httpRequestsMockHelpers.setLoadPipelinesResponse(pipelines);
```

Comment:
nit: this can be declared once in the `describe` block as it is not being overridden.